# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1333691484 25200
#      Thu Apr 05 22:51:24 2012 -0700
# Node ID 2946642797d6ed9f60d73937076d129d31ca2e34
# Parent  8985cbbd2fe4b7fd1a8362c34d35ab7be09fa9a9
maint: remove execute permission bits on splinefit functions.

* __splinefit__.m, splinefit.m: remove execute permission bits.

diff --git a/scripts/polynomial/private/__splinefit__.m b/scripts/polynomial/private/__splinefit__.m
old mode 100755
new mode 100644
diff --git a/scripts/polynomial/splinefit.m b/scripts/polynomial/splinefit.m
old mode 100755
new mode 100644
